eclipse/sw360
# fix(UI): Default behavior of write access checkbox restore

Signed-off-by: Nikesh kumar <kumar.nikesh@simens.com>

[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> Please provide a summary of your changes here.

we have restored the default behavior of write access checkbox.

> * Which issue is this pull request belonging to and how is it solving it? (*Refer to issue here*)
> * Did you add or update any new dependencies that are required for your change?

Issue: 
we are not able test rest api  (post,patch) url from postman,To test these kind of url we need write access.

### Suggest Reviewer
> You can suggest reviewers here with an @mention.

### How To Test?
1: login to sw360
2: Go to preferences tab
3: scroll down and we can see write access checkbox. 

> How should these changes be tested by the reviewer?
> Have you implemented any additional tests?

### Checklist
Must:
